prelude/toolchains/cxx.bzl

```diff
@@ -27,7 +27,7 @@ def _system_cxx_toolchain_impl(ctx: AnalysisContext):
     """
     A very simple toolchain that is hardcoded to the current environment.
     """
-    archiver_args = ["ar", "rcs"]
```

Comment:
This seems like a good idea independently. I might try and land that separately, so that we have a revertable lump for each piece.